ui/app/templates/vault/cluster/secrets/backend/roles.hbs

```diff
@@ -1,143 +0,0 @@
-<PageHeader as |p|>
-  <p.top>
```

Comment:
This file was not being used anywhere. Double checked in the test suite and the router.